spec/controllers/idv/hybrid_mobile/entry_controller_spec.rb

```diff
@@ -35,15 +35,6 @@
         get :show, params: { 'document-capture-session': 'foo' }
       end
 
-      it 'logs an analytics event' do
-        expect(@analytics).to have_logged_event(
```

Comment:
It looks like we've removed all analytics assertions from this file. We can remove `stub_analytics` from the `before` block above (line 18) if we're not expecting this controller to log any analytics.